README.md

```diff
@@ -44,5 +44,5 @@ A curation of plugins, prompts, and resources for the [friendly interactive shel
 - [Abbreviation Tips](https://github.com/Gazorby/fish-abbreviation-tips) - Remembering abbreviations by displaying tips when you can use them.
 
 ## Docker
-
+- [Alpine containers](https://hub.docker.com/r/andreiborisov/fish)
```

Comment:
Container or containers?

Reply:
I guess we'd keep it in line with the existing entry

Reply:
👍